Content.Shared/Damage/Systems/PassiveDamageSystem.cs

```diff
@@ -37,18 +37,42 @@ public override void Update(float frameTime)
             if (comp.NextDamage > curTime)
                 continue;
 
-            if (comp.DamageCap != 0 && damage.TotalDamage >= comp.DamageCap)
+            // Check if there are any passive damage entries.
+            if (comp.PassiveDamageList.Count == 0)
                 continue;
 
```

Comment:
Re: `DamageSum`, can put that local variable here:
```suggestion

var damageSum = new DamageSpecifier();

```